go.mod

```diff
@@ -6,6 +6,8 @@ replace cloud.google.com/go => cloud.google.com/go v0.102.1
 
 replace github.com/vmware-tanzu/tanzu-cli/test/e2e/framework => ./test/e2e/framework
 
+replace github.com/vmware-tanzu/tanzu-plugin-runtime => github.com/anujc25/tanzu-plugin-runtime v0.0.0-20240214002508-cc83ffc09fda
```

Comment:
Update once https://github.com/vmware-tanzu/tanzu-plugin-runtime/pull/163 is merged.